### PR TITLE
Added Script to convert Megatron LM to . nemo file

### PR DESCRIPTION
**Pull request by yidong72**
Tested with 345M Bert and 5B GPT checkpoints. Works fine.

### CODE REVIEW

File: examples/nlp/language_modeling/megatron_lm_pt_to_nemo.py [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+
+def get_args():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--checkpoint_folder",
+        type=str,
+        default=None,
+        required=True,
+        help="Path to PTL checkpoints saved during training. Ex: /raid/Megatron_LM/checkpoints",
```
Comment (ericharper) [inline examples/nlp/language_modeling/megatron_lm_pt_to_nemo.py:51]: I think there's a typo here, it should be "Path to Megatron-LM checkpoints saved ..." ?

**Review by ericharper — CHANGES_REQUESTED** [review]
Could you add a Jenkins test that converts a small megatron-lm checkpoint to .nemo?

File: examples/nlp/language_modeling/megatron_lm_pt_to_nemo.py [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""
+Conversion script to convert Megatron_LM checkpoints into nemo checkpoint.
+  Example to run this conversion script:
+    python -m torch.distributed.launch --nproc_per_node=<tensor_model_parallel_size> megatron_lm_pt_to_nemo.py.py \
```
Comment (ericharper) [inline examples/nlp/language_modeling/megatron_lm_pt_to_nemo.py:18]: Could we change the name of the script to `megatron_lm_ckpt_to_nemo.py` for consistency? Also typo here `.py.py`

File: examples/nlp/language_modeling/megatron_lm_pt_to_nemo.py (hunk 1 above)
Comment (yidong72) [reply examples/nlp/language_modeling/megatron_lm_pt_to_nemo.py:51]: fixed

File: examples/nlp/language_modeling/megatron_lm_pt_to_nemo.py (hunk 2 above)
Comment (yidong72) [reply examples/nlp/language_modeling/megatron_lm_pt_to_nemo.py:18]: fixed

**Comment by yidong72** [comment]
Added the Jenkins test case.

**Review by ericharper — DISMISSED** [review]
LGTM. Thanks!

**Pull request merged**